src/daemon/utils_cache.c

```diff
@@ -787,7 +790,8 @@ int uc_get_history(const data_set_t *ds, const value_list_t *vl,
   return uc_get_history_by_name(name, ret_history, num_steps, num_ds);
 } /* int uc_get_history */
 
-int uc_get_hits(const data_set_t *ds, const value_list_t *vl) {
+int uc_get_hits(__attribute__((unused)) const data_set_t *ds,
```

Comment:
We could just remove the `ds` argument. There are only two call sites.

Reply:
done